Makefile

```
@@ -228,28 +228,44 @@ endif
 
 api: dev-db ## API server on the configured port; assets under ./data (make deps first)
+	@set -a; \
+	if [ -f "$(CURDIR)/deploy/compose/.env" ]; then . "$(CURDIR)/deploy/compose/.env"; fi; \
+	set +a; \
 	cd backend && STORAGE_LOCAL_PATH=$(CURDIR)/data \
 		ALLOWED_ORIGINS=http://localhost:$(WEB_PORT) \
 		PUBLIC_URL=http://localhost:$(API_PORT) \
+		FLEET_TOKEN_KEY="$${FLEET_TOKEN_KEY:-$$FLEET_SECRET}" \
 		BENCHMARK_API=1 TELEMETRY=false .venv/bin/uvicorn app.main:app --port $(API_PORT)
 
 worker-rocm: ## inference worker on the AMD GPU (make setup-rocm once)
```

checkmake (0.3.2)

[warning] Target body for "worker-rocm" exceeds allowed length of 5 lines (9).

(maxbodylength)